README.md

```diff
@@ -15,10 +15,11 @@ Proposals follow [the TC39 process](https://tc39.github.io/process-document/) an
 * [Active Proposals](https://github.com/tc39/proposals/blob/master/ecma402/README.md)
 * [Stage 0 Proposals](https://github.com/tc39/proposals/blob/master/ecma402/stage-0-proposals.md)
 
-### Contributing New Proposals
+## How to Contribute
 
 Please see [Contributing to ECMAScript](/CONTRIBUTING.md) for the most up-to-date information on contributing proposals to this standard.
 
+ECMA 402 is developed by the ECMA 402 Task Group under TC39.  The task group achieves consensus and brings recommendations to TC39.  The task group has a monthly teleconference.  Community members who have followed the procedures in [CONTRIBUTING.md](/CONTRIBUTING.md) are welcome to join the monthly meeting.  Please see the [ECMA 402 Monthly Calendar](https://calendar.google.com/calendar/embed?src=unicode.org_nubvqveeeol570uuu7kri513vc%40group.calendar.google.com) for the schedule and for information on how to dial into the meeting.  Please also apply for membership in the [ecma402-delegates@chromium.org](https://groups.google.com/a/chromium.org/forum/#!forum/ecma402-delegates) mailing list.
```

Comment:
for formality and convention, let's use ECMA-402 only.

We can also formally refer to the ECMA-402 TG as TC39-TG2 but may not be necessary here.